contrib/macdeploy/README.md

```diff
@@ -12,6 +12,8 @@ When complete, it will have produced `Bitcoin-Core.dmg`.
 
 ### Step 1: Obtaining `Xcode.app`
 
+An Apple Developer Account is required to proceed.
```

Comment:
Maybe: "A free Apple Developer Account" (to clarify that you don't need to sign up for their paid app developer program)